### DISCUSSION TITLE
Datasette to process webhooks

### DISCUSSION THREAD

**Discussion by bt-maps** [simonw/datasette#1714] [Q&A]
Hi, great project. I am looking to use datasette with  spatialite to  store and visualize json and geojson geoms from a webhook. The response looks similar to this.
{
  "command": "set",
  "group": "5c5203ccf5ec4e4f349fd038",
  "detect": "inside",
  "hook": "warehouse",
  "key": "fleet",
  "time": "2019-01-30T13:06:36.769273-07:00",
  "id": "bus",
  "object": { "type": "Point", "coordinates": [-112.26, 33.46] }
}

I see that there has been some recent work write to db functionality. Hoping I can get some guidance on best practice to implement this in datasette 
Thanks
Brent

**Comment by bt-maps** [comment]
https://github.com/mfa/datasette-webhook-write

This seems to be similar to what I am looking for.

**Comment by nileshtrivedi** [comment]
It would be interesting to see a plugin for provide JSON APIs to do database writes.
In fact, projects like PostgREST, Supabase and Datasette could align on the API interface for JSON writes for convenient frontend development.

**Reply by brandonrobertz** [reply]
This would be really straightforward to implement using sqlite_utils' JSON insert and some really basic token credentials in the metadata for auth.
